handbook/company/pricing-features-table.yml

```diff
@@ -97,17 +118,56 @@
           moreInfoUrl: https://docs.google.com/document/d/1qFYtMoKh3zyERLhbErJOEOo2me6Bc7KOOkjKn482Sqc/edit
         - description: See what servers someone is logged-in on
           moreInfoUrl: https://community.spiceworks.com/topic/138171-is-there-a-way-to-see-what-servers-someone-is-logged-in-on
+    - industryName: Automated export/sync
+      friendlyName: Build custom query automations
+      description: Ship logs with snapshots of any imaginable report, or monitor results for changes.
+      tier: Free
+      usualDepartment: Security
+      waysToUse:
+        - description: Ship logs to Splunk, Snowflake, and more
+        - description: Synchronize live state of endpoints to a data lake or SIEM in a consistent shape.
+        - description: Export the data to other systems
+          moreInfoUrl: https://docs.google.com/document/d/1pE9U-1E4YDiy6h4TorszrTOiFAauFiORikSUFUqW7Pk/edit
+        - description: Export data to a third-party SIEM tool
+          moreInfoUrl: https://www.websense.com/content/support/library/web/hosted/admin_guide/siem_integration_explain.aspx
 - categoryName: Integrations
   features:
     - industryName: REST API
-      friendlyName: Automate any feature  
-      description: ""
+      friendlyName: Automate any feature
+      description: 
       documentationUrl: https://fleetdm.com/docs/rest-api/rest-api 
-      screenshotSrc: ""
+      screenshotSrc:
       tier: Free
       dri: rachaelshaw
     - industryName: Command line tool (CLI)
+      friendlyName: fleetctl
+      tier: Free
+    - industryName: Webhooks
+      friendlyName: 
       tier: Free
+    - industryName: Deep automations
+      friendlyName: Trigger webhooks or run scripts
+      description: Fire off webhooks or run scripts on hosts when certain things happen in Fleet.
+      productCategories: [Endpoint operations,Device management,Vulnerability management]
+      comingSoonOn: 2024-06-30
```

Comment:
jeff abbot, will mayhone

Reply:
@noahtalerman @willmayhone88 https://github.com/fleetdm/confidential/issues/3996  « that is an ugly stub, but hopefully enough to communicate the idea